server/webhook/jobs.go

```diff
@@ -38,11 +38,12 @@ func (w *webhook) handleChannelJob(ctx context.Context, event *gitlab.JobEvent)
 	default:
 		return res, nil
 	}
-	message += fmt.Sprintf("**Repository**: [%s](%s)\n", event.ProjectName, event.Repository.GitHTTPURL)
+	namespace, project := normalizeNamespacedProjectByHomepage(event.Repository.Homepage)
+	fullNamespacePath := fmt.Sprintf("%s/%s", namespace, project)
+	message += fmt.Sprintf("**Repository**: [%s](%s)\n", fullNamespacePath, event.Repository.GitHTTPURL)
```

Comment:
Probably doesn't matter because it forwards to the browser-friendly URL, but I think this is supposed to be used:

```suggestion
	message += fmt.Sprintf("**Repository**: [%s](%s)\n", fullNamespacePath, event.Repository.HTTPURL)
```